packages/dev/core/src/Rendering/IBLShadows/iblShadowsVoxelTracingPass.ts

```diff
@@ -343,7 +344,11 @@ export class _IblShadowsVoxelTracingPass {
 
         this._frameId++;
 
-        let rotation = this._scene.useRightHandedSystem ? -(this._envRotation + 0.5 * Math.PI) : this._envRotation - 0.5 * Math.PI;
+        let rotation = 0.0;
+        if (this._scene.environmentTexture) {
+            rotation = (this._scene.environmentTexture as CubeTexture).rotationY;

```

Comment:
```suggestion
            rotation = (this._scene.environmentTexture as CubeTexture).rotationY ?? 0;
```
just in case the environment texture isn't a cube (I don't know if this is possible, but the change is easy enough).